libs/cart/state/src/actions/cart.actions.ts

```diff
@@ -109,6 +110,10 @@ export class DaffResolveCartFailure implements Action {
   constructor(public payload: string) {}
 }
 
+export class DaffResolveCartServerSide implements Action {
```

Comment:
This special case action should definitely have docs.

Reply:
I've added a tutorial doc as well as class docs.